app/Resources/doc/sysadmin.md

```diff
@@ -0,0 +1,141 @@
+saves the item information and remains on the **Edit** page, while **Update and close** saves the
+information and returns to the list page. The **Delete** button deletes the item after asking for
+confirmation.
+
+![Edit user page buttons](images/editUserPageButtons.png)
+
+###Deleting items
+
+Note that **Country**, **Site**, and **User** items have an **Active** checkbox. These items cannot
+be deleted if there still exist case records with references to them, so the ability to deactivate
+them allows their use to be discontinued for future case records.
+
+Items can be deleted individually by clicking the item's **Delete** button in the **Action** column
+of the list page or the **Delete** button below the item information in the **Edit** page.
+
+Items can be deleted in bulk by selecting the checkboxes next to their names in the list page,
+selecting the **Delete** action at the bottom of the page, and clicking **OK**.
+
+**CAUTION: Enabling the All elements checkbox at the bottom of the list will apply the Delete
+**command to all items in the section, *including those that are not currently visible due to
+**filtering or pagination.*
+
+User Specific
+-------------
+
+###Access Restrictions
+
+The **Access Restrictions** section of the **Create** or **Edit** user page can be used to limit
+user access to specific categories of cases or the ability to import and export data for specific
+areas. The **Add new** button below the Access Restrictions list is used to create one or more
+restrictions that apply to the user.
+
+![Access Restrictions](images/accessRestrictions.png)
+
+Use the **Type** drop-down to select a category on which to limit access.
+
+Click the **Options**
+field to select one or more actions that the user will be allowed to perform on the **Target**. An
+option you have already added can be removed if necessary by clicking the **x** next to it. Leaving
+the **Options** field empty will give the user view-only access to the **Target**.
+
+Begin typing the specific criterion under **Target**. Matching criteria from the relevant internal
+list will be offered as autocomplete suggestions.
+
+Logically, each Access Restriction should be thought of as an *allow only* limitation. To allow
+access to multiple specific criteria in a given category, add additional restrictions.
+
+**Access Restrictions** can be deleted by enabling the **Delete** checkbox next to them and then
+saving the user information with a **Create** or **Update** button.
+
+Reference Labs Specific
+-----------------------
+
+You can select multiple countries in the **Countries** drop-down list. Select one at a time, then
+click in the field again to select another. Click the **x** next to any country that you want to
+remove from the field.
+
+Import Maps Specific
+--------------------
+
+Please refer to import.md for full documentation of import/export functions.
+
```

Comment:
can you make the refer to link to that? Should be possible similar to the inline images